lab_1/main.py

```diff
@@ -62,7 +113,17 @@ def compare_profiles(unknown_profile: dict, profile_to_compare: dict, top_n: int
     :param top_n: a number of the most common words
     :return: the distance
     """
-    pass
+    if not isinstance(unknown_profile, dict) or not isinstance(profile_to_compare, dict) or \
+            not isinstance(top_n, int):
+        return None
+    top_unknown_profile = get_top_n_words(unknown_profile['freq'], top_n)
+    top_profile_to_compare = get_top_n_words(profile_to_compare['freq'], top_n)
+    common_words = []
+    for profile_word in top_profile_to_compare:
```

Comment:
explain what happens in this loop, please

Reply:
get the tops of the words of two profiles, take each profile_word in top_profile_to_compare, if the profile_word is in top_unknown_profile, add it to the common_words.divide amount of elements in the list of common_words by list of top words and round it.